frontend/src/app/testQueue/TestQueue.tsx

```diff
@@ -92,7 +92,6 @@ export const queueQuery = gql`
           number
         }
       }
-      result
```

Comment:
⚠️ Didn't want to do too much in this PR but eventually these queries should be moved to its respective `operations.graphql` file